_data/accessibility-tests/breadcrumb.yml

```diff
@@ -0,0 +1,125 @@
+title: Breadcrumb
+component_status: pass
+test_items:
+# General tests
+  - summary: Breadcrumb text and background meet color contrast requirements. 
+    summary_additional: |
+      When you use a color contrast checker to view the breadcrumb navigation, 
+      contrast between the text and background colors is at least 4.5:1. 
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.3
+  - summary: Breadcrumb separators (arrows) meet color contrast requirements.    
+    summary_additional: |
+      When you use a color contrast checker to view the breadcrumb navigation, 
+      contrast between the separators (arrows) and background colors is at least 3:1. 
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.11
+  - summary: Breadcrumb links appear in the same location and order on every page of the website.
+    summary_additional: |
+      When you navigate to different pages on the website, the breadcrumb links are 
+      always in the same order and in the same place.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 3.2.3
```

Comment:
**todo:** This test should be listed as **conditional**. The test status on the left of the spreadsheet (🔒) seems to have fallen out of sync with it's latest results column.

```suggestion
  - summary: Breadcrumb links appear in the same location and order on every page of the website.
    summary_additional: |
      When you navigate to different pages on the website, the breadcrumb links are 
      always in the same order and in the same place.
    test_status: conditional
    test_type: general
    version_tested: 3.8.2
    wcag_criterion: 3.2.3
```